packages/features/bookings/Booker/config.ts

```diff
@@ -54,25 +54,29 @@ export const resizeAnimationConfig: ResizeAnimationConfig = {
   small_calendar: {
     default: {
       width: "calc(var(--booker-meta-width) + var(--booker-main-width))",
+      // maxWidth: "100%",
       gridTemplateAreas: `"meta main"`,
       gridTemplateColumns: "var(--booker-meta-width) var(--booker-main-width)",
     },
     selecting_time: {
-      width: "calc(var(--booker-meta-width) + var(--booker-main-width) + var(--booker-timeslots-width))",
+      width: "100%",
+      maxWidth: "calc(var(--booker-meta-width) + var(--booker-main-width) + var(--booker-timeslots-width))",
       gridTemplateAreas: `"meta main timeslots"`,
-      gridTemplateColumns: "var(--booker-meta-width) var(--booker-main-width) var(--booker-timeslots-width)",
+      gridTemplateColumns: "var(--booker-meta-width) 1fr var(--booker-timeslots-width)",
```

Comment:
This actually is the real improvement here. The 1fr instead of a fixed width allows it to scale down and fit in the remaining space.